MT/VR-MT-0006/tests/test0001.json

```diff
@@ -0,0 +1,389 @@
+{
+  "name": "2/2 and validation a = 90 days later",
+  "payload": {
+    "v": [
+      {
+        "dt": "2022-01-01",
+        "dn": 2,
+        "sd": 2
+      }
+    ]
+  },
+  "expected": true,
+  "external": {
+    "validationClock": "2022-04-01T00:00:00Z",
+    "valueSets": {
```

Comment:
I tend to whittle the `external` section down to what's needed minimally. That avoids big Git diffs, at the cost of having to think about whether this could influence the result.